python/py-slackclient/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        slackapi python-slackclient 2.7.2 v
+name                py-slackclient
+categories-append   irc
+platforms           darwin
```

Comment:
the package does not install architecture-dependent files, so please add the line `supported_archs noarch`